### ISSUE TITLE
"Welcome to Civic Tech"? page/tab

### ISSUE THREAD

**Issue by DesignsbyZay** [Code4Newark/codefornewark.org#9]
- Civic Tech 101 video embedded
- Digital literacy and Civic Tech Basics resources: local and free online

**Comment by BradleyGonmiah** [comment]
Link to "Civic Hacking 101": https://www.youtube.com/watch?v=wH6LnW_qjeI

**Comment by davidruffner** [comment]
File to change is `pages/new-hackers.md`

**Comment by davidruffner** [comment]
Progress! See #36 #34

**Comment by 21mooie** [comment]
I've been thinking that this page could be used to engage a new member to a codefornewark hack session and give them an introduction to topics such as 

- GitHub

- version control

- web development

- text editors/IDE

and this would be interactive so all the new members could learn together what this means and how the codefornewark team uses them to build applications to help the city. This way they could be brought up to speed in a way so that in future meetings they can help build the applications currently being developed by the team.

**Comment by davidruffner** [comment]
I agree @21mooie, those are all great things to add. I think it would be good to include a brief description of civic hacking 101 as well.

**Comment by 21mooie** [comment]
Ok. I'll jot some quick ideas I have for this and see what you think.

**Comment by 21mooie** [comment]
I wrote down a process and ran my parents through it to see how good it operates. I'll open a pull request soon to show the changes and get y'all opinion.

**Comment by 21mooie** [comment]
#37 I've added this PR.

**Comment by 21mooie** [comment]
@davidruffner I've been thinking more about what you said in terms of making the hacker page more general. Would this mean some kind of structure that asks how the user would like to be involved with codefornewark and then directs them to resources that explains that information for them.
